### ISSUE TITLE
Incompatible With ItemSwapper (1.19.2)

### ISSUE THREAD

**Issue by Jazztache** [PieKing1215/InvMove#46]
**Describe the bug**
Whilst using InvMove with the ItemSwapper mod, the quick swap menu stops working. After isolating InvMove and ItemSwapper, I noted that holding down the item swapper disables your movement, and only works when standing still.

ItemSwapper: https://modrinth.com/plugin/itemswapper

**To Reproduce**
1. Install both InvMove and ItemSwapper on the same Minecraft instance
2. Hold down the swap key and movement keys at the same time, attempting to switch whilst doing so.

**Screenshots**
See attached an mp4 video file. I had to scale it down to 620p to make it fit.

https://github.com/PieKing1215/InvMove/assets/97380337/3d311577-d34b-472e-abbb-952f6e653b6b

**Version and Mod List**
List the version of InvMove, the version of Forge/Fabric/Quilt, and the version of Minecraft the issue happens on:
InvMove: 0.4.0-mc1.19.2
Forge/Fabric/Quilt version: 0.15.7
Minecraft: Release 1.19.2
PrismLauncher: 8.0-1

List any other mods being used:
ItemSwapper
Fabric API
Cloth Config v8
Fabric Language Kotlin
No Prying Eyes
No Chat Reports
No Telemetry

**Comment by rollercobester** [comment]
I was actually able to reproduce this on 1.20.1, although it's a three mod issue from what I can tell.

When using ItemSwapper with InvMove AND MouseWheelie, I kept getting that flash and movement stopping when pressing R. But if I removed either InvMove or MouseWheelie, it worked just fine.

**Comment by Jazztache** [comment]
I did have MouseWheelie in that footage! Thankyou for the clarification! Still intending to use MouseWheelie though...
Side note: just wondering if there is any precedent for three-way mod compatibility issues? A lot of MC Mod repos list mod comptibility with one other mod at a time, but not two.

**Comment by PieKing1215** [comment]
Yeah this was a weird issue, but it'll be fixed in the next InvMove release
I still don't really understand why MouseWheelie (Amecs library specifically) was needed for the issue to happen but whatever
